GVRf/Framework/src/org/gearvrf/GVRViewManager.java

```diff
@@ -667,7 +675,9 @@ public void beforeDrawEyes() {
 
             doMemoryManagementAndPerFrameCallbacks();
 
-            mScript.onStep();
+            GVRViewManager.this.getEventManager().sendEvent(
```

Comment:
It exits from line 88 in GVREventManager.java. If the function is not defined in the script, GVRScriptFile.invokeFunction(...) internally receives a ScriptException and then it returns false. Since nothing is wrong in this case, the function sendEvent(..) exits normally.